app/views/asset/shares/ShareClassView.scala.html

```diff
@@ -35,7 +35,7 @@
 
     @formHelper(action = ShareClassController.onSubmit(index, draftId), 'autoComplete -> "off") {
 
-        @errorSummary(form.errors)
+        @errorSummary(form.errors, ShareClass.allOptions)
 
         @inputRadio(
```

Comment:
Sorry I didn't notice it the first time but this radio is missing `inline = false`

I'm not sure why it's not at a component level since really only Yes/No radios are typically inline and that's a separate component... but you can either add the line here or set the default at the component level.

And then it'll be good to merge at last!

Reply:
Good spot